### PR TITLE
[13.0][MIG] base_import_security_group: Migration to 13.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 13.0

Please @pedrobaeza an @joao-p-marques can you review it?

@Tecnativa TT29357

### CODE REVIEW

File: base_import_security_group/security/base_import_security_group_security.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <!-- GROUPS -->
+    <record id="group_import_csv" model="res.groups">
+        <field name="name">Import CSV/Excel files</field>
+        <field name="category_id" ref="base.module_category_hidden" />
```
Comment (chienandalu) [inline base_import_security_group/security/base_import_security_group_security.xml]: Otherwise is shown as a dropdown in the user config
```suggestion
```

**Review by chienandalu — COMMENTED** [review]
Check tests

**Comment by StefanRijnhart** [comment]
Duplicate of https://github.com/OCA/server-ux/pull/279, so closing this one. Feel free to reopen if you manage to fix the javascript tests!

**Comment by victoralmau** [comment]
Tests work fine now, can you review it?

**Review by StefanRijnhart — APPROVED** [review]
Excellent, thank you for rewriting the tests!

**Review by hbrunn — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]

**Pull request merged**